### ISSUE TITLE
how to change the font?

### ISSUE THREAD

**Issue by zwyyy456** [luizdepra/hugo-coder#720]
I have read [ How to change font? #696 ](https://github.com/luizdepra/hugo-coder/issues/696), and I changed `$font-family` in `/themes/hugo-coder/assets/scss/_variable.scss`, but the font in the page seems don't change.
This is `font-family` changed by me for test:
```
$font-family: "Microsoft YaHei", "微软雅黑";

```

**Comment by zwyyy456** [comment]
It works after I create new `_variable.scss` under `assets/scss/` and override the override the original`_varibale.scss` in the theme. Thanks a lot for your excellent hugo theme and your reply!

**Comment by dharmit** [comment]
@luizdepra if I understand correctly, one has to create a `assets/scss` directory in the root of the project for this and then create a `_variables.scss` file under it, right?

**Comment by luizdepra** [comment]
Correct, @dharmit.
This way Hugo will override the original file.

**Comment by dharmit** [comment]
OK. I tried the same. Weirdly, it works fine on localhost:1313, but the deploy preview URL generated for https://github.com/dharmit/dharmitshah/pull/22 seems to be ignoring it. What could be the issue? 🤔

**Comment by luizdepra** [comment]
Sometimes this kind of problem is related to the Hugo's Assets Pipeline that is only present in the Extended binary.
Check if the PR preview is being generated with the extended version.

But, I think if you run `hugo server` and just `hugo` on local host, and upload all generated resource files, it will work too. I'm not 100% sure.

**Comment by dharmit** [comment]
You're right about extended binary. I didn't notice that on localhost I had been using extending, while not so in the GitHub action that builds deploy preview URL.

Thanks again! :)
